mcgill-en.csl

```diff
@@ -37,6 +37,18 @@
         <single>para</single>
         <multiple>paras</multiple>
       </term>
+      <term name="sub verbo" form="short">
+			  <single></single>
```

Comment:
The sub-verbo thing was a work-around from a couple of years for cases 
where some other form of locator was to be added in by the user. The 
biggest one was "article." Because there is no locator for Article, 
references to article need to be inserted directly by the user, and 
without any leading text that might be inserted by another term. Users 
of the McGill guide are almost certain not use sub verbo.  Does that 
make sense?

On 2015-12-12 9:33 PM, Sebastian Karcher wrote:
>
> In mcgill-en.csl 
> <https://github.com/citation-style-language/styles/pull/1826#discussion_r47436413>:
>
> > @@ -37,6 +37,18 @@
> >          <single>para</single>
> >          <multiple>paras</multiple>
> >        </term>
> > +      <term name="sub verbo" form="short">
> > +			  <single></single>
>
> what's the idea here?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/citation-style-language/styles/pull/1826/files#r47436413>.
>

-- 
Liam McHugh-Russell, JD, LLM
Researcher, Department of Law
European University Institute, Villa Schifanoia
Via Boccaccio, 121
I - 50133 Firenze, Italy
+39 33 4811 4340